pcap4j-core/src/main/java/org/pcap4j/packet/IcmpV6HomeAgentAddressDiscoveryReplyPacket.java

```diff
@@ -0,0 +1,266 @@
+         * +                      Home Agent Addresses                     +
+         * |                                                               |
+         * +                                                               +
+         * |                                                               |
+         * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+         */
+        private static final long serialVersionUID = 7184228144196703852L;
+
+        private static final int IDENTIFIER_OFFSET = 0;
+        private static final int IDENTIFIER_SIZE = SHORT_SIZE_IN_BYTES;
+        private static final int RESERVED_OFFSET = IDENTIFIER_OFFSET + IDENTIFIER_SIZE;
+        private static final int RESERVED_SIZE = SHORT_SIZE_IN_BYTES;
+        private static final int HOME_AGENT_ADDRESSES_OFFSET = RESERVED_OFFSET + RESERVED_SIZE;
+        private static final int HOME_AGENT_ADDRESSES_SIZE = INET6_ADDRESS_SIZE_IN_BYTES;
+        private static final int ICMPV6_HOME_AGENT_ADDRESS_DISCOVERY_REPLY_HEADER_MIN_SIZE = HOME_AGENT_ADDRESSES_OFFSET
+                + HOME_AGENT_ADDRESSES_SIZE;
+
+        private final short identifier;
+        private final short reserved;
+        private final List<Inet6Address> homeAgentAddresses;
+
+        private IcmpV6HomeAgentAddressDiscoveryReplyHeader(byte[] rawData, int offset, int length)
+                throws IllegalRawDataException {
+            if (length < ICMPV6_HOME_AGENT_ADDRESS_DISCOVERY_REPLY_HEADER_MIN_SIZE) {
+                StringBuilder sb = new StringBuilder();
+                sb.append("The data is too short to build an ICMPv6 Home Agent Address Discovery Reply Header(")
+                        .append(ICMPV6_HOME_AGENT_ADDRESS_DISCOVERY_REPLY_HEADER_MIN_SIZE)
+                        .append(" bytes). data: ")
+                        .append(ByteArrays.toHexString(rawData, " "))
+                        .append(", offset: ")
+                        .append(offset)
+                        .append(", length: ")
+                        .append(length);
+                throw new IllegalRawDataException(sb.toString());
+            }
+            this.identifier = ByteArrays.getShort(rawData, IDENTIFIER_OFFSET + offset);
+            this.reserved = ByteArrays.getShort(rawData, RESERVED_OFFSET + offset);
+            this.homeAgentAddresses = new ArrayList<Inet6Address>();
+            for (int i = HOME_AGENT_ADDRESSES_OFFSET; i < length; i += INET6_ADDRESS_SIZE_IN_BYTES) {
+                homeAgentAddresses.add(ByteArrays.getInet6Address(rawData, i + offset));
+            }
+        }
+
+        private IcmpV6HomeAgentAddressDiscoveryReplyHeader(Builder builder) {
+            this.identifier = builder.identifier;
+            this.reserved = builder.reserved;
+            this.homeAgentAddresses = builder.homeAgentAddresses;
+        }
+
+        public short getIdentifier() {
+            return identifier;
+        }
+
```

Comment:
Please add a method that returns the identifier as int like below:

```java
public int getIdentifierAsInt() {
  return identifier & 0xFFFF;
}
```

Reply:
added.